src/crypto/crypto_mbedtls_alt.c

```diff
@@ -916,7 +920,7 @@ int omac1_aes_256(const u8 *key, const u8 *data, size_t data_len, u8 *mac)
 
 #endif /* MBEDTLS_CMAC_C */
 
-#ifdef MBEDTLS_AES_C
+#if defined(MBEDTLS_AES_C) || defined(CONFIG_PSA_WANT_ALG_CTR) || defined(CONFIG_PSA_WANT_KEY_TYPE_AES)
```

Comment:
why add CTR here?

Reply:
Earlier, I had both CBC check and Counter check but later replaced it by AES check. I agree, CTR can be removed here. I will do so